uc-crux-llvm/src/UCCrux/LLVM/Config.hs

```diff
@@ -84,16 +86,26 @@ ucCruxLLVMConfig = do
               Crux.yesOrNoSpec
               False
               "Re-explore functions that have already been explored (i.e., have logs)"
-            <*> Crux.section
-              "entry-points"
-              (Crux.listSpec Crux.stringSpec)
-              []
-              "Comma-separated list of functions to examine."
             <*> Crux.section
               "explore-budget"
               Crux.numSpec
               8
               "Budget for exploration mode"
+            <*> Crux.section
+              "explore-timeout"
+              Crux.numSpec
+              5
+              "Hard timeout for exploration of a single function (seconds)"
+            <*> Crux.section
+              "explore-parallel"
+              Crux.yesOrNoSpec
+              False
+              "Explore different functions in parallel"
+            <*> Crux.section
+              "entry-points"
+              (Crux.listSpec Crux.stringSpec)
+              []
```

Comment:
Is there an implicit `main` default?

Reply:
No, it will suggest you use `--entry-points` or `--explore` if neither is provided.